plugins/exec.lua

```diff
@@ -11,6 +11,19 @@ local function exec(cmd, keep_newline)
 end
 
 
+local function exec_rw(dv, cmd, keep_newline)
+  local proc = process.start { "sh", "-c", cmd }
```

Comment:
Might be a good idea to check that this didn't fail. In that case a `core.error` message might be appropriate.

Reply:
Done